package.json

```diff
@@ -64,8 +67,14 @@
     "@babel/plugin-proposal-object-rest-spread": "7.10.1",
     "@babel/plugin-transform-runtime": "7.10.1",
     "@babel/preset-env": "7.10.2",
+    "@babel/preset-typescript": "^7.12.17",
     "@babel/register": "7.10.1",
     "@babel/runtime": "^7.10.2",
+    "@types/bignumber.js": "^5.0.0",
```

Comment:
proposing to remove this package because of warning:
> npm WARN deprecated @types/bignumber.js@5.0.0: This is a stub types definition for bignumber.js (https://github.com/MikeMcl/bignumber.js/). bignumber.js provides its own type definitions, so you don't need @types/bignumber.js installed!